addons/web/static/tests/services/notification_service_tests.js

```diff
@@ -52,7 +52,7 @@ QUnit.module('Services', {
         await testUtils.nextMicrotaskTick();
         var $notification = $('body .o_notification_manager .o_notification');
         assert.strictEqual(_.str.trim($notification.html().replace(/\s+/g, ' ')),
-            "<div class=\"toast-header\"> <span class=\"fa fa-2x mr-3 fa-lightbulb-o o_notification_icon\" role=\"img\" aria-label=\"Notification undefined\" title=\"Notification undefined\"></span> <div class=\"d-flex align-items-center mr-auto font-weight-bold o_notification_title\">a</div> </div> <div class=\"toast-body\"> <div class=\"o_notification_content\">b</div> </div>",
```

Comment:
Why has this to be changed ?

Reply:
It's strange I know I can't explain why but the tests were failing due to the tag properties being in a different order after having the notification.xml file loaded via the manifest. Maybe by being loaded this way the t-att properties are loaded in a certain order. I can investigate but I don't think that is relevant.

Reply:
@fja-odoo Mmh, probably because of this which is not needed anymore I suppose: https://github.com/odoo/odoo/commit/b94395b50175925fc7d1bd3655e5facb78a8ba2b#diff-6f28813339836ad86da5eb4475180f22R106, to adapt

Reply:
why not needed ? what is there to adapt ? All I see is that previously when notification were in `base.xml` the tag properties were in the same order as it is now with `notification.xml` being loaded like `base.xml` wich confirm my changes no ?

Reply:
> @fja-odoo Mmh, probably because of this which is not needed anymore I suppose: [b94395b#diff-6f28813339836ad86da5eb4475180f22R106](https://github.com/odoo/odoo/commit/b94395b50175925fc7d1bd3655e5facb78a8ba2b#diff-6f28813339836ad86da5eb4475180f22R106), to adapt

I added this, if you do not lazy load anymore, this is to remove...